src/commands/junit/renderer.ts

```diff
@@ -27,8 +29,26 @@ export const renderRetriedUpload = (payload: Payload, errorMessage: string, atte
   return chalk.yellow(`[attempt ${attempt}] Retrying jUnitXML upload ${jUnitXMLPath}: ${errorMessage}\n`)
 }
 
-export const renderSuccessfulCommand = (fileCount: number, duration: number) =>
-  chalk.green(`${ICONS.SUCCESS} Uploaded ${fileCount} files in ${duration} seconds.\n`)
+export const renderSuccessfulCommand = (fileCount: number, duration: number, spanTags: SpanTags) => {
+  let fullStr = ''
+  fullStr += chalk.green(`${ICONS.SUCCESS} Uploaded ${fileCount} files in ${duration} seconds.\n`)
+
+  const testRunsUrl = getTestRunsUrl(spanTags)
+  if (testRunsUrl) {
+    fullStr += chalk.green(
+      '=================================================================================================\n'
+    )
+    fullStr += chalk.green(
+      '* View test runs and detailed time reports on Datadog (can take a few minutes to become available)\n'
```

Comment:
what do you mean by detailed time reports?

Reply:
I copied what we're showing for `dogweb` jobs. Totally opened to change it if there is a better copy 👌🏼

Reply:
Could we say, "View the detailed test report for this commit on Datadog (this may take a few minutes to become available)"

And this is where the link takes them? To the Commit Overview page? Or does it take them somewhere else?